packages/client/src/platform.js

```diff
@@ -10,5 +11,7 @@ export {
   Headers,
   Blob,
   File,
-  Blockstore
+  Blockstore,
+  filesFromPath,
+  getFilesFromPath
```

Comment:
What is the difference?

Reply:
`getFilesFromPath` iterates the `filesFromPath` async iterator and returns an array with all the files. I think users will likely rely on the `getFilesFromPath`, but in the CLI we do a transformation so the iterator is also nice for such use cases.